test_hm24/test_hm24.py

```diff
@@ -0,0 +1,73 @@
+"""Homework 24 - saucedemo.com"""
+
+import pytest
+from hm_24 import (login_page, main_page_add,
+                   main_page_remove, cart_page_delete_el,
+                   menu, log_out)
+
+
+def test_login_page(test_data, driver_got):
+    """
+    1. Экран логина:
+    1.1 Ввод имени пользователя:
+    1.2 Ввод пароля - password
+    1.3 Нажатие кнопки 'Login':  #login-button
+    """
+    res = login_page(test_data[0], test_data[1], driver_got)
+    assert res, "Error here"
```

Comment:
Please, define better error message
```suggestion
    assert res, "Error here"
```